examples/nlp/gpt/conf/gpt_reinforce_actor.yaml

```diff
@@ -0,0 +1,218 @@
+defaults:
+  - optional tp_overlap@model.ub_tp_comm_overlap_cfg:
+
+trainer:
+  # these args are respected
+  num_nodes: 8
+  devices: 8
+  accelerator: gpu
+  precision: bf16
+
+  reinforce:
+
+    max_epochs: 1
+    max_steps: -1  # max REINFORCE steps (-1 to go through the whole train set)
+    val_check_interval: 10
+    save_interval: ${.val_check_interval}
+    gradient_clip_val: 1.0
+
+    # REINFORCE args to generate the data for training
+    initial_policy_kl_penalty: 0.01
+    use_absolute_kl: True
+    num_rollouts_per_prompt: 4
+
+
+    # the sequence length to pad the rollout batch for training to
+    # reduce fragmentation at the cost of using more
+    # memory, set to null if we don't want to pad it
+    # to a constant size
+    # if actual seq length is higher than this a warning will be raised
+    # but will not crash and training will still proceed on the larger
+    # sequence length
+    rollout_batch_seq_length: null
+
+    # Speed-up training by accelerating inference stage using TRTLLM
+    trt_llm:
+      enable: False
+      reshard: False # if True then reshard the model into TP only for inference
+
+      # TRTLLM preallocates activation memory according to the number of input tokens
+      # By default, assume the max input length is the difference between the model sequence length and the max number of tokens to generate
+      max_input_len: ${subtract:${model.encoder_seq_length}, ${model.reinforce.length_params.max_length}}
+      max_input_tokens: ${multiply:${.max_input_len}, ${model.reinforce.rollout_micro_batch_size}}
```

Comment:
should be okay to drop, but please double check

Reply:
Dropped!